docs/deployment.md

```diff
@@ -26,7 +26,7 @@ Production URL is [foundation.mozilla.org](https://foundation.mozilla.org)
 
 ### Domain Redirect
 
-Enable domain redirection by setting `DOMAIN_REDIRECT_MIDDLWARE_ENABLED` to `True`. This will enable a middleware function that checks every request, and return a 307 redirect to `TARGET_DOMAIN` if the host header does not match it.
+Enable domain redirection by setting `DOMAIN_REDIRECT_MIDDLWARE_ENABLED` to `True`. This will enable a middleware function that checks every request, and return a 307 redirect to the first listed domain in `TARGET_DOMAINS` if the host header does not match one of the domains specified there.
```

Comment:
typo we never caught: `DOMAIN_REDIRECT_MIDDLWARE_ENABLED` should be `DOMAIN_REDIRECT_MIDDLEWARE_ENABLED`